lib/scarpe/glibui/alert.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
```

Comment:
we can ignore everything but para for this pr. but I'm keeping these as prompts for the next prs.